force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="slds-grid slds-theme_default slds-grid--align-spread">
+
+        <div class="slds-col slds-m-left_x-small">
+            <span>{ComponentTitle}</span>
+        </div>
+
+        <div class="slds-col">
+            <span>
+                <div if:true={editButtonShown}>	
+                    <lightning-button label={labelReference.settingsButtonEdit} title="Edit" onclick={handleEditClick} class="slds-p-right_x-small"></lightning-button>	
+                </div>	
+                <div if:false={editButtonShown}>	
+                    <lightning-button label={labelReference.settingsButtonCancel} title="Cancel" onclick={handleCancelClick} class="slds-m-left_x-small"></lightning-button>
+                    <lightning-button variant="brand" label={labelReference.settingsButtonSave} title="Primary action" onclick={handleSaveClick} class="slds-p-right_x-small"></lightning-button>
```

Comment:
Let's use the custom label for the titles too. We may need more for A11y concerns.
```suggestion
                    <lightning-button label={labelReference.settingsButtonCancel} title={labelReference.settingsButtonCancel} onclick={handleCancelClick} class="slds-m-left_x-small"></lightning-button>
                    <lightning-button variant="brand" label={labelReference.settingsButtonSave} title={labelReference.settingsButtonSave} onclick={handleSaveClick} class="slds-p-right_x-small"></lightning-button>
```